bindata/network/ovn-kubernetes/ovnkube-node.yaml

```diff
@@ -105,7 +105,9 @@ spec:
           iptables -t raw -A OUTPUT -p udp --dport {{.GenevePort}} -j NOTRACK
           retries=0
           while true; do
-            db_ip=$(kubectl get ep -n ${ovn_config_namespace} ovnkube-db -o jsonpath='{.subsets[0].addresses[0].ip}')
+            # wait 30s for kubectl get to return
+            # TODO: change to use '--request-timeout=30s', if https://github.com/kubernetes/kubernetes/issues/51952 is fixed. 
```

Comment:
oops, meant to comment here: the first part ("wait 30s for kubectl get to return") doesn't seem necessary (it's pretty obvious from either "timeout 30" or "--request-timeout=30s"). For the second part, that issue was closed as a duplicate of 49343, so link there instead?

Reply:
fixed.